template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/AzureAd/AzureAdServiceCollectionExtensions.cs

```diff
@@ -15,20 +15,19 @@ namespace Microsoft.AspNetCore.Authentication.Extensions
 {
     public static class AzureAdServiceCollectionExtensions
     {
-        public static IServiceCollection AddAzureAdAuthentication(this IServiceCollection services)
+        public static IServiceCollection AddAzureAd(this IServiceCollection services)
```

Comment:
This should now be an extension on AuthenticationBuilder:

So the startup will look like:
AddAuthentication().AddAzureAd()